.travis.yml

```diff
@@ -1,6 +1,9 @@
-language: java
+language:
+ - java
+ - python
 jdk:
   - openjdk8
+python: "3.6"
```

Comment:
the only change that's needed to fix the travis python error is to change line 10 to..
 
```
before_install:
        - "pyenv install 3.6.3"
        - "pyenv global 3.6.3"
```

reference: https://github.com/apiaryio/dredd/issues/1448

Reply:
Thanks!